deploy/git/mixer_hash.txt

```diff
@@ -0,0 +1 @@
+f0a283b
```

Comment:
please revert this. it should be gitignored, can you check why it's getting added